lib/model.js

```diff
@@ -340,7 +344,29 @@ ModelBaseClass.prototype._initProperties = function(data, options) {
       }
     }
   }
-  this.trigger('initialize');
+  this.__trigger('initialize');
+};
+
+ModelBaseClass.prototype.__trigger = function trigger(actionName, work, data, callback) {
```

Comment:
I wanted to confirm with you whether I could just remove the trigger function and call afterinitialize directly, but I wasn't sure if we'll be adding anything in the future. I think we can remove that, do you agree?